.github/workflows/tests.yml

```
@@ -0,0 +1,57 @@
+name: "Run Backend Tests"
+
+on:
+  push:
+    branches: [ "main" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)